### DISCUSSION TITLE
Who manages the app in the F-Droid repository?

### DISCUSSION THREAD

**Discussion by AxeldeWater** [JunkFood02/Seal#91] [General]
I am completely new to GitHub and stuff and barely know anything about coding. I liked this project though and want to contribute, and already did what I could: I translated the app to Dutch. Now I wondered who manages the app when it comes to the F-Droid repository, if that is a difficult task, how it works and if this could maybe be something I could do. I came to wonder all this because I read about v1.0.0 being released two days ago, but it still not being on F-Droid.

**Comment by JunkFood02** [comment]
Thanks for your contribution! I know little about how F-Droid works but for sure it's managed automatically. F-Droid will build and release their own apks, so it'll have a delay of around 2-4 days for a new version on F-Droid after I release it here.

**Reply by AxeldeWater** [reply]
Okok understood, great! And no problem!
